.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+name: CI
+on: [pull_request]
+
+env:
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1
```

It's recommended now to use the v2 cache, as the v1 didn't cache quite as well, and resulted in a lot of misses.

```suggestion
      - uses: actions/cache@v2
```